app/helpers/grantees_helper.rb

```diff
@@ -0,0 +1,5 @@
+module GranteesHelper
```

Comment:
[suggestion] although we discussed not doing FakeData in this change, since it's done and you included a Grantee model, we don't need this helper, the grantee name is accessible directly from the model

Reply:
👍🏾